src/megaapi_impl.cpp

```diff
@@ -431,7 +439,8 @@ MegaNode* MegaNodePrivate::getPublicNode()
 
     MegaNode *node = new MegaNodePrivate(
                 name, type, size, ctime, mtime,
-                plink->ph, &key, &attrstring, fingerprint);
+                plink->ph, &key, &attrstring, fingerprint,
+                INVALID_HANDLE);
```

Comment:
`INVALID_HANDLE` is the default value of this constructor. Specifying it here is redundant.